src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1015,6 +1027,13 @@ def compute_several_extensions(self, extensions, save=True, verbose=False, **job
             It the extension can be saved then it is saved.
             If not then the extension will only live in memory as long as the object is deleted.
             save=False is convenient to try some parameters without changing an already saved extension.
+        verbose : bool, default: False
+            If True, print more information about the computation.
+        force_recompute : bool, default: False
+            If True, the extension is recomputed even if it has already been computed with the provided parameters.
+            If False, the extension is not recomputed if it has already been computed with the provided parameters.
+        **job_kwargs : keyword arguments
+            All other kwargs are transmitted to job_kwargs
```

Comment:
```suggestion
            All other keyword arguments are transmitted to job_kwargs
```

I still have a slight preference to say keyword arguments here because even though kwarg is well known in the python world, it is not apparent to new python users (ie a big chunk of users). So giving people a chance to kwarg and keyword argument together will help them make the connection in the future.